### ISSUE TITLE
Missing airport Saratov-Gagarin (IATA GSV)

### ISSUE THREAD

**Issue by mauriciojost** [opentraveldata/opentraveldata#118]
From wikipedia:  "operations will commence in Q2 2019" (https://en.wikipedia.org/wiki/Saratov_Gagarin_Airport GSV).

**Comment by denisarnaud** [comment]
Thanks for the report, Mauricio!
I'll try to take of it later in the week

**Comment by mauriciojost** [comment]
@denisarnaud maybe you could instruct us on how to contribute so that we can help you? It's the operation performed manual? Do you use a specific script to generate the new line? As you may know ( ;) ) we use this dataset a lot.

**Comment by denisarnaud** [comment]
Thanks for your contribution!

Yes, do not hesitate to submit pull requests, so as to streamline their adoption in the OPTD project. The whole process has extensively been documented in the [README file of the `tools` directory](https://github.com/opentraveldata/opentraveldata/tree/master/tools/README.md).

There are basically two steps:
1. Daily step (to be performed once a day, at max), described in the [Geonames-derived POR file section](https://github.com/opentraveldata/opentraveldata/tree/master/tools#geonames-derived-por-file), explaining how to download and pre-process Geonames points of reference (POR) data, and how to extract the POR data for OPTD (temporary `por_{intorg,all}_YYYYMMDD.csv` files, stored in the [`tools` directory](https://github.com/opentraveldata/opentraveldata/tree/master/tools) once generated).
2. Iterative step, to be performed every time POR details are altered in OPTD-curated data files ([`optd_por_best_known_so_far.csv`](http://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_best_known_so_far.csv) and/or [`optd_por_no_longer_valid.csv`](http://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_no_longer_valid.csv)), which re-generates the POR CSV data files ([`optd_por_public.csv`](https://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_public.csv) and [`optd_por_public_all.csv`](http://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_public_all.csv)).

In other words:
1. Only [`optd_por_best_known_so_far.csv`](http://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_best_known_so_far.csv) and/or [`optd_por_no_longer_valid.csv`](http://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_no_longer_valid.csv) should be manually edited
2. [`optd_por_public.csv`](https://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_public.csv) and [`optd_por_public_all.csv`](http://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_public_all.csv) should be re-generated by the [`make_optd_por_public.sh`](https://github.com/opentraveldata/opentraveldata/blob/master/tools/make_optd_por_public.sh) script
